applications/HDF5Application/python_scripts/core/operations/model_part.py

```diff
@@ -299,104 +325,132 @@ class PrimalBossakOutput(VariableIO):
     transient adjoint solvers.
     '''
 
-    def __init__(self, settings):
-        super(PrimalBossakOutput, self).__init__(settings)
-        settings.SetDefault('alpha_bossak', -0.3)
-        self.alpha_bossak = settings['alpha_bossak']
+    def __init__(self,
+                 model_part: KratosMultiphysics.ModelPart,
+                 parameters: KratosMultiphysics.Parameters,
+                 file: KratosHDF5.HDF5File):
+        self.__alpha_bossak = parameters["alpha_bossak"].GetDouble()
+        super().__init__(model_part, parameters, file)
+
+    @property
+    def alpha_bossak(self) -> float:
+        return self.__alpha_bossak
 
-    def __call__(self, model_part, hdf5_file):
-        primal_io = KratosHDF5.HDF5NodalSolutionStepBossakIO(
-            self.GetSettings(model_part).Get(), hdf5_file)
-        primal_io.SetAlphaBossak(self.alpha_bossak)
-        primal_io.WriteNodalResults(model_part)
+    def Execute(self) -> None:
+        primal_io = KratosHDF5.HDF5NodalSolutionStepBossakIO(self.parameters, self.file)
+        primal_io.SetAlphaBossak(self.__alpha_bossak)
+        primal_io.WriteNodalResults(self.model_part)
 
+    @staticmethod
+    def GetDefaultParameters() -> KratosMultiphysics.Parameters:
+        parameters = super(PrimalBossakOutput, PrimalBossakOutput).GetDefaultParameters()
```

Comment:
```suggestion
        parameters = super().GetDefaultParameters()
```

Reply:
I need to explicitly pass the class type because `GetDefaultParameters` is a `staticmethod` and thus cannot implicitly provide `self` for `super` (though it'd be much cleaner if `GetDefaultParameters` was instead a `classmethod` so I'll make that change).